package/dotnet/package

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(
+    dotnet-profile
+    dotnet-host
+    dotnet-sdk
+    dotnet-runtime
+    aspnet-runtime
+    dotnet-targeting-pack
+    aspnet-targeting-pack
+    netstandard-targeting-pack
+)
+pkgver=3.1.10-1
+timestamp=2020-12-27T18:48Z
+maintainer="Eeems <eeems@eeems.email>"
+url=https://www.microsoft.com/net/core
+section=utils
+license=MIT
+
+source=(
+    https://download.visualstudio.microsoft.com/download/pr/2ebe1f4b-4423-4694-8f5b-57f22a315d66/4bceeffda88fc6f19fad7dfb2cd30487/dotnet-sdk-3.1.404-linux-arm.tar.gz
+    dotnet-profile
+)
+sha256sums=(
+    c95e8cf72abbcde1ac3974158ed9355d8d5588bade463fdf69d6932f915f453a
+    SKIP
+)
+
+dotnet-profile() {
+    pkgdesc="Default profile script to ensure that the .NET Core Command Line Interface runs properly"
+
+    package() {
+        install -Dm 744 -t "$pkgdir"/etc/profile.d/ "$srcdir"/dotnet-profile
+    }
+    configure() {
+        echo "Make sure to source /etc/profile.d/dotnet-profile before running dotnet"
+    }
+}
+
+dotnet-host() {
+    pkgdesc="A generic driver for the .NET Core Command Line Interface"
+    depends=(dotnet-profile)
+
+    package() {
+        install -dm 755 "$pkgdir"/opt/usr/{bin,lib,share/{dotnet,licenses/dotnet-host}}
+        cp -dr --no-preserve='ownership' "$srcdir"/{dotnet,host} "$pkgdir"/opt/usr/share/dotnet/
+        cp -dr --no-preserve='ownership' "$srcdir"/{LICENSE,ThirdPartyNotices}.txt "$pkgdir"/opt/usr/share/licenses/dotnet-host
+        ln -sf /opt/usr/share/dotnet/dotnet "$pkgdir"/opt/usr/bin/dotnet
```

Comment:
Maybe use `/opt/bin` instead? Our bootstrap still does not add `/opt/usr/bin/` to the path. So most users will not find the dotnet command.